### ISSUE TITLE
logo exact size

### ISSUE THREAD

**Issue by plegrand1** [CatimaLoyalty/Android#1374]
Hello,
Just a very simple question:
What should be the exact perfect size of the logo of each entry?
Thanks
Pascal

**Comment by obfusk** [comment]
AFAIK the cards in the app are the [same dimensions as physical banking and ID cards](https://en.wikipedia.org/wiki/ISO/IEC_7810#ID-1): 85.6 × 53.98 (mm).

**Comment by TheLastProject** [comment]
Yup, exactly what obfusk said. All card logos in the app are scaled to exactly that size: https://github.com/CatimaLoyalty/Android/blob/6937342226b56ef9104aab5df7a749f70f99d246/app/src/main/res/layout/loyalty_card_view_layout.xml#LL55C3-L55C3 :)

**Comment by obfusk** [comment]
And when selecting an image, you are offered the option to crop it to that size as well :)

**Comment by plegrand1** [comment]
Hello and thank you very much for your answers.
Is there a "place" where I can get these logos in the right size?
Thank you again.

**Comment by obfusk** [comment]
> Is there a "place" where I can get these logos in the right size?

I haven't used any thumbnails myself, so not sure. But you could take a photo of the physical card. Or try to find an example of one online. Or find an image of the store logo and crop that to the right size.

**Comment by plegrand1** [comment]
I downloaded some logos from the net and resized them (85.6mm × 53.98mm). 
But every time I insert them in the application, they are "truncated".

**Comment by plegrand1** [comment]
![Screenshot-Leclerc](https://github.com/CatimaLoyalty/Android/assets/8543044/eb86d4b8-241f-44ea-a63b-4e46bb731e90)
![leclerc](https://github.com/CatimaLoyalty/Android/assets/8543044/2873a39c-0929-4e4a-b59d-140dd39db220)

**Comment by plegrand1** [comment]
Excellent !!!
I followed your advice to create a blank image at the right size.
Perfect!!!

**Comment by ukanuk** [comment]
I'd appreciate if the 856:5398 ratio was shown somewhere so I can crop my photos accordingly. Maybe in the FAQ https://catima.app/faq/ or in the app itself when user clicks CARD ratio instead of 1:1 or ORIGINAL. But if most people don't care as much as me about giving Catima an image already having the exact right ratio, I can certainly understand not wanting to put it in.
